test/image.bats

```diff
@@ -5,6 +5,12 @@ load helpers
 IMAGE=quay.io/crio/pause
 SIGNED_IMAGE=registry.access.redhat.com/rhel7-atomic:latest
 UNSIGNED_IMAGE=quay.io/crio/hello-world:latest
+IMAGE_LIST_DIGEST=docker.io/library/alpine@sha256:7746df395af22f04212cd25a92c1d6dbc5a06a0ca9579a229ef43008d4d1302a
+IMAGE_LIST_DIGEST_AMD64=docker.io/library/alpine@sha256:bf1684a6e3676389ec861c602e97f27b03f14178e5bc3f70dce198f9f160cce9
```

Comment:
are we sure this digest won't change?

Reply:
The list's digest is affected by the contents, so these two won't unless the tag is moved, in which case the list's digest would change, but the per-arch images might not (for example, if an image for a new arch was added to the list, which would change the list's digest, and the tag was moved to point to the new list).